pkg/controller/issueapi/issue_batch.go

```diff
@@ -72,7 +72,7 @@ func (c *Controller) HandleBatchIssue() http.Handler {
 			return
 		}
 
-		if !membership.Can(rbac.CodeBulkIssue) {
+		if membership.User != nil && !membership.Can(rbac.CodeBulkIssue) {
```

Comment:
I think you need to check if _membership_ is nil. Membership.User will never be nil.

Reply:
Yep sorry. Users have RBAC, not AuthApps.